src/client/app/components/meters/EditMeterModalComponent.tsx

```diff
@@ -240,6 +243,22 @@ export default function EditMeterModalComponent(props: EditMeterModalComponentPr
 				props.meter.maxError != state.maxError ||
 				props.meter.disableChecks != state.disableChecks
 			);
+		let error_message = '';
+		for (const value of Object.values(groups)) {
```

Comment:
I think this check is run every time. Is that correct? For example, if there is no change should it run? Another is if you only change the name then does it matter if it is in another group? I suspect this should be restricted to certain cases. Sorry if the issue and our conversations did not cover this and let me know if you think otherwise.

Reply:
i changed the implementation so that it only runs when it needs to. What do you think?

Reply:
I also restricted it more.